tests/functional/ext/keyword/missing_keyword_arg.py

```diff
@@ -0,0 +1,96 @@
+"""Tests for consider-using-keyword-argument"""
+# pylint: disable=unused-argument, missing-function-docstring, missing-class-docstring,
+# pylint: disable=invalid-name, too-few-public-methods, line-too-long, too-many-arguments
```

Comment:
```suggestion
# pylint: disable=too-few-public-methods, line-too-long, too-many-arguments
```

It's often nice to use "Pythonic" names in the tests as well as they can be used by non-maintainers are reference to what the checker does. Making the tests "pythonic" helps there I think.